.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,6 +5,9 @@ for guidance on how to set up your development environment,
 run the test suite, write new integrations, and more.
 -->
 
+**Change log entry**
+<!-- If this is a customer-visible change, a brief summary to be placed into the change log. -->
+
```

Comment:
I like this BUT should we say something for community contributors? Something along the lines of:

> `<!-- For Datadog folks, if this is a customer-visible change, a brief summary to be placed into the change log. -->`

or

> `<!-- If this is a customer-visible change, a brief summary to be placed into the change log. (Feel free to ask for help on this one) -->`

...something like that?

Reply:
A contributed PR could be customer-visible (e.g., it could fix a bug). What about language to the effect of "If you are not a Datadog employee and you are not sure what to do, please leave this section in the PR but do not fill it out"?

Reply:
Yeap, that sounds good! I agree with you that a community PR may also be part of the changelog, my concern was only in making it clear in the instructions what non-datadog employees should fill in that section :)

Reply:
Created https://github.com/DataDog/dd-trace-rb/pull/4021 with the above change